apps/docs/content/components/Prose.mdx

```diff
@@ -0,0 +1,110 @@
+    <blockquote>Nulla ac odio eu magna hendrerit porta. Donec nec eros quis tortor tincidunt vulputate. Aenean id pharetra diam, sit amet auctor leo. Aliquam erat volutpat.</blockquote>
+    <p>Integer pellentesque pretium nulla viverra molestie. Praesent quis pretium sapien. Sed convallis eget lectus et pulvinar:</p>
+    <ul>
+      <li>
+        Vivamus eu risus nec lectus consequat rutrum at vel lacus.
+      </li>
+      <li>Donec at dolor ut metus imperdiet congue vel porta nunc.
+      </li>
+      <li>Quisque eu tortor suscipit, congue quam in, bibendum tellus.</li>
+    </ul>
+    <h3>Heading level 3</h3>
+    <p>Pellentesque non ornare ligula. Suspendisse nibh purus, pretium id tortor sit amet, tincidunt gravida augue. Ut malesuada, nisl vel dignissim mollis</p>
+    <img
+      src="https://via.placeholder.com/400x400/f5f5f5/f5f5f5.png"
+      alt="placeholder, blank area with an off-white background color"
+      />
+    <h4>Heading level  4</h4>
+    <p>
+      Secure code as you write it. Automatically review every change to your codebase and identify vulnerabilities
+      before they reach production. <a href="/#">Learn more here.</a>
+    </p>
+    <h5>Heading level 5</h5>
+    <ol>
+      <li>
+        Vivamus eu risus nec lectus consequat rutrum at vel lacus.
+      </li>
+      <li>Donec at dolor ut metus imperdiet congue vel porta nunc.
+      </li>
+      <li>Quisque eu tortor suscipit, congue quam in, bibendum tellus.</li>
+    </ol>
+    <h6>Heading level 6</h6>
+    <p>Pellentesque non ornare ligula. Suspendisse nibh purus, pretium id tortor sit amet, tincidunt gravida augue.</p>
+    <p>Nunc velit odio, posuere eu felis eget, consectetur fermentum nisi. Aenean tempor odio id ornare ultrices. Quisque blandit condimentum tellus, semper efficitur sapien dapibus nec. </p>
+    `}
+/>
+```
+
+### Full Width
+
+```jsx live
+<Prose
+  enableFullWidth
+  html={`
+	<h1>The tools you need to build what you want.</h1>
+    <p>Contribute to projects quickly with automatic environment setup.</p>
+    <p>Make sure you see the changes you care about.</p>
+    <p>Build community around your code.</p>
+
+    <img
+      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
+      alt="placeholder, blank area with an off-white background color"
+    />
+
+    <h2>Automation and CI/CD</h2>
+    <ul>
+      <li>
+        Automate everything: CI/CD, testing, planning, project management, issue labeling, approvals, onboarding, and
+        more.
+      </li>
+      <li>Standardize and scale best practices, security, and compliance across your organization.</li>
+      <li>Get started quickly with thousands of actions from partners and the community.</li>
+    </ul>
+
+    <h2>Security</h2>
+    <p>
+      Secure code as you write it. Automatically review every change to your codebase and identify vulnerabilities
+      before they reach production. <a href="/#">Learn more here.</a>
+    </p>
+`}
```

Comment:
The content in this example feels quite heavy for illustrating the full width prop, it's also the same as the content we had in the default example which has since been replaced. Recommend making this html more compact to redirect user attention to the prop.

```suggestion
<Prose
  enableFullWidth
  html={`
    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut tempor lectus ipsum, consectetur convallis diam pretium quis. Proin ut felis ut eros tristique tincidunt.</p>

    <p>Nulla ac odio eu magna hendrerit porta. Donec nec eros quis tortor tincidunt vulputate. Aenean id pharetra diam, sit amet auctor leo. Aliquam erat volutpat.</p>
    `}
```